jerry-core/lit/lit-magic-strings.inc.h

```diff
@@ -219,6 +219,7 @@ LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_G_CHAR, "g")
 LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_I_CHAR, "i")
 LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_M_CHAR, "m")
 LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_TIME_SEP_U, "T")
+LIT_MAGIC_STRING_DEF (LIT_MAGIC_STRING_TIME_ZONE_U, "Z")
```

Comment:
"Z" means UTC time. I think the name should contains it. Maybe `LIT_MAGIC_STRING_UTC_TIME_ZONE_U`?